Tests/FlatGammaCaloSteps.fcl

```diff
@@ -0,0 +1,5 @@
+#include "Production/JobConfig/primary/FlatGammaCalo.fcl"
+source.firstRun: 501
```

Comment:
This run number isn’t inside the valid MDC2020 range and the results will not be processable (digitization and reconstruction) due to lack of conditions data. I’m not sure if that matters for this test.

Reply:
Is there a better run number to use for testing? I took this from an existing test script.

Reply:
you need something 1200-1210

Reply:
But don't use 1205, that's for the extracted position.

Reply:
I updated the test fcl to use run 1210 instead